src/core/utils.js

```diff
@@ -451,7 +451,7 @@ export function runTransforms(content, flist) {
 class NetworkError extends Error {
   constructor(message, response) {
     super(message);
-    Object.defineProperty(this, "response", { get() { return response; } });
+    Object.defineProperty(this, "response", { value: response });
```

Comment:
as it's readonly, would we want `writable: false` also?

Reply:
Yup, and it's false by default 👍

Reply:
ah, sorry, I'd assumed it was true by default (that would make no sense). Thanks for clarifying!

Reply:
Given the discussion in the other bug about just using standard fetch error handling, maybe we don't need this at all.

Reply:
Agreed, closing in favor of standard error handling.